app/views/layouts/application.html.erb

```diff
@@ -4,7 +4,7 @@
     <title><%= render 'layouts/title' %></title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="manifest" href="manifest.json">
+    <link rel="manifest" href="/manifest.json">
```

Comment:
Did you double-check that this links correctly?

Reply:
I was actually not sure about this one.
When I ran the command "blc http://localhost:3000 -ro", it didn't show any broken link (like the above screenshot).
How do I correct it?

Reply:
We don't need to change this link! I checked and what's already there works in production.